app/services/taxonomy/taxon_unpublisher.rb

```diff
@@ -31,7 +33,10 @@ def tag_to_parent(taxon_content_id, user)
 
     def unpublish_taxon(taxon_content_id, redirect_to_content_id)
       redirect_to_taxon = Services.publishing_api.get_content(redirect_to_content_id)
-      Services.publishing_api.unpublish(taxon_content_id, type: "redirect", alternative_path: redirect_to_taxon["base_path"])
+      publishing_api_unpublish(taxon_content_id, redirect_to_taxon["base_path"])
```

Comment:
I wonder if it would be simpler to just copy/paste the previous line and surround with an 'if' block, like we do [here](https://github.com/alphagov/content-tagger/pull/1000/files#diff-63a8e971bcba960dd75982227cab40e0R178). Although normally we'd aim to reduce duplication, in this case I think it's more important to keep the changes minimal and additive, so they're easy to identify.

Reply:
Same linting error as below 😞

Reply:
I'd actually be happy if we disabled the cop for this purpose, but it's not a biggie.